backend/migrations/20240524154034_incident_data.sql

```diff
@@ -0,0 +1,20 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE incident_data (
+    instance_id VARCHAR(255) NOT NULL,
+    incident_id VARCHAR(255) NOT NULL,
```

Comment:
We can make this as a unique column, and rename it simply as `id`

Reply:
i don't think incident_id should be unique, since i want it to be used to group several incident data together, to make a historical view like in other platform

Reply:
or do you have any other tought about this?

Reply:
If you want to group it, you can do a select query with the compound index of `monitor_id` and `title`. Title shouldn't change, right?

Reply:
okay, then incident_id isn't need right? since it can be grouped by title i think if we want to have a detailed look on the api we can just query by title and status or timestamp.